### ISSUE TITLE
What is this mysterious `.` file that gets uploaded

### ISSUE THREAD

**Issue by peterbe** [mdn/yari#1529]
Not sure where it comes from. If it's our builder or our uploader. 
<img width="1398" alt="Screen Shot 2020-10-26 at 10 34 06 AM" src="https://user-images.githubusercontent.com/26739/97186371-97303c80-1777-11eb-8106-36231b1b8958.png">

When you click to download it, you get this:
<img width="1139" alt="Screen Shot 2020-10-26 at 10 34 21 AM" src="https://user-images.githubusercontent.com/26739/97186430-a7e0b280-1777-11eb-8123-e51d45dce5a2.png">

Is that what's actually in the file or is that a generic S3 error message?

**Comment by peterbe** [comment]
I ran a clean build locally. Then I ran the uploader to my personal AWS S3 account. And the file certainly got created there too. Hmm...

**Comment by peterbe** [comment]
@escattone I figured it out! It's the file `PosixPath('../client/build/index.html')`
I figured it out by running:
```python
    def upload(self, bucket_manager):
        # print("FILE", repr(str(self.key)))
        if str(self.key) == "main/.":
            print("KEY:", repr(self.key))
            print("FILE_PATH", repr(self.file_path))
            raise Exception("FILE FREAK OUT")
```

Now, the question is, what should we do with it?

**Comment by peterbe** [comment]
Thing is, at the moment, nobody should be viewing this file via CloudFront so my instinct tells me ought to be excluded. 
With Yari owning the "homepage" or not, that URL is actually `/en-US` not `/`. The URL `/` has a dedicated special behavior in CF/Lambda that *always* return a `302 Found`. Never a file from S3. 

In Yari1, we intend to make the URL `/en-US` actually pass through to Kuma Django which returns a body of HTML. 
The day Yari takes over the home page (and other things like the sign-up and site-search), the actual URL is going to be `/en-US` which would equate to the Yari builder creating the file `client/build/en-us/index.html` which our `deployer` should correctly treat as:
```python
s3_client.upload_file(
  '../client/build/en-us/index.html', 
  bucket_name, 
  'PREFIX/en-us`
)
```

So, in conclusion, the root `index.html` is useless for uploading to S3. We should quite simply skip this file.
